packages/hosted-form-v2/src/iframe-content/hosted-input.ts

```diff
@@ -145,6 +145,7 @@ export default class HostedInput {
 
             case 'card-name':
             case 'note':
+            case 'hidden':
```

Comment:
We need a new kind of hosted form field to submit the payment form without collecting user-input data.
So creating a new `hidden` type like `note`, but its form value is not required.